sql_generators/glean_usage/templates/baseline_clients_yearly_v1.query.sql

```diff
@@ -0,0 +1,78 @@
+{{ header }}
+
+{% if init %}
```

Comment:
Why do we do this init step? Does the regular artifact deployment process not deploy these generated tables?

Reply:
No, these are created by the `bqetl query backfill` command, which runs `initialize` if the table doesn't exist yet. (the issue is that since this is self-referencing, it can't get the schema by dry-running the query since the query won't run!)